src/xscen/io.py

```diff
@@ -510,24 +509,33 @@ def _skip(var):
 
         if mode == "o":
             if exists:
-                var_path = path / var
-                msg = f"Removing {var_path} to overwrite."
-                logger.warning(msg)
-                sh.rmtree(var_path)
+                clean_incomplete(path, incomplete=[var])
```

Comment:
pas si grave et I see that it does the same, but the name is a bit confusing. Using mode='o' is not necessarily because it was incomplete.